src/packages/src/xroad/common/base/usr/share/xroad/scripts/_setup_db.sh

```diff
@@ -75,8 +75,11 @@ setup_database() {
   local db_addr=${hosts[0]%%:*}
   local db_port=${hosts[0]##*:}
 
-  local_psql() { su -l -c "psql -qtA -p ${db_port:-5432} $*" postgres; }
-  remote_psql() { psql -h "${db_addr:-127.0.0.1}" -p "${db_port:-5432}" -qtA "$@"; }
+  db_addr="${db_addr:-127.0.0.1}"
+  db_port="${db_port:-5432}"
```

Comment:
Is it required to have these 2 lines?

I mean, `db_add` and `db_port` are extracted from `db_hosts` which defaults to `127.0.0.1:5432` already (lines [28](https://github.com/nordic-institute/X-Road/pull/2242/files?diff=unified&w=1#diff-3ba8dd19685852c1e16e3055e629c9a99b475a2151edc08955f6218c93beb938R28) and [74](https://github.com/nordic-institute/X-Road/pull/2242/files?diff=unified&w=1#diff-3ba8dd19685852c1e16e3055e629c9a99b475a2151edc08955f6218c93beb938R74))

Or, am I missing something?